site/_posts/2017-09-29-miqldap-to-sssd.md

```diff
@@ -0,0 +1,315 @@
+---
+title: "MiqLdap To SSSD conversion tool"
+author: jvlcek
+date: 2017-09-29
+comments: true
+published: true
+tags: tutorials
+---
+
+How does one user the *miqldap_to_sssd* conversion tool?
```

Comment:
Should it be "How does one use"? (not user)

Reply:
👍